packages/fxa-payments-server/src/components/PaymentMethodHeader/index.test.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import { PaymentMethodHeader } from '.';
+import {
+  getLocalizedMessage,
+  MOCK_PLANS,
+  setupFluentLocalizationTest,
+} from '../../lib/test-utils';
+import { act } from 'react-dom/test-utils';
```

Comment:
I don't think this is the `act` we need; it should be from @testing-library/react